### PR TITLE
CAMEL-10641: Load Camel TypeConverters when a new TypeConverterLoader…

### PR DESCRIPTION
**Pull request by johnpoth**
… OSGI service is detected

https://issues.apache.org/jira/browse/CAMEL-10641

Note https://github.com/apache/camel/pull/1371 is needed to run the tests.

### CODE REVIEW

**Comment by johnpoth** [comment]
Yeah sorry about that, fixed :)

File: tests/camel-itest-osgi/src/test/resources/org/apache/camel/itest/TypeConverter [hunk 1]
```diff
@@ -0,0 +1 @@
+org.apache.camel.itest.typeconverter.MyConverter
```
Comment (davsclaus) [inline tests/camel-itest-osgi/src/test/resources/org/apache/camel/itest/TypeConverter]: Not to be picky, but we should have an ASF license header in these files too. You can copy from any existing similar file name.

Before we do releases we run a RAT check which would detect this problem

**Comment by davsclaus** [comment]
Thanks for the PR it has been merged. Do you mind closing this?

**Comment by johnpoth** [comment]
Thanks for looking into this :)

**Pull request closed without merging**